tests/unit/test_workflow.py

```diff
@@ -442,3 +442,71 @@ def test_chaining_3():
     assert all(
         x in result.columns for x in ["ad_id_count", "ad_id_clicked_sum_ctr", "ad_id_clicked_sum"]
     )
+
+
+@pytest.mark.parametrize("shuffle", [nvt.io.Shuffle.PER_WORKER, nvt.io.Shuffle.PER_PARTITION, None])
+@pytest.mark.parametrize("use_client", [True, False])
+@pytest.mark.parametrize("apply_offline", [True, False])
+def test_workflow_apply(client, use_client, tmpdir, shuffle, apply_offline):
+    out_files_per_proc = 2
+    out_path = str(tmpdir.mkdir("processed"))
+    path = str(tmpdir.join("simple.parquet"))
+
+    size = 25
+    row_group_size = 5
+
+    cont_columns = ["cont1", "cont2"]
+    cat_columns = ["cat1", "cat2"]
+    label_column = ["label"]
+
+    df = pd.DataFrame(
+        {
+            "cont1": np.arange(size),
+            "cont2": np.arange(size),
+            "cat1": np.arange(size),
+            "cat2": np.arange(size),
+            "label": np.arange(size),
+        }
+    )
+    df.to_parquet(path, row_group_size=row_group_size, engine="pyarrow")
+
+    dataset = nvt.Dataset(path, engine="parquet", row_groups_per_part=1)
+    processor = nvt.Workflow(
+        cat_names=cat_columns,
+        cont_names=cont_columns,
+        label_name=label_column,
+        client=client if use_client else None,
+    )
+    processor.add_cont_feature([ops.FillMissing(), ops.Clip(min_value=0), ops.LogOp()])
+    processor.add_cat_preprocess(ops.Categorify())
+
+    processor.finalize()
+    # Force dtypes
+    dict_dtypes = {}
+    for col in cont_columns:
+        dict_dtypes[col] = np.float32
+    for col in cat_columns:
+        dict_dtypes[col] = np.float32
```

Comment:
Just checking - Is the default type different than `"float32"` for all columns? (I want to make sure we are checking that all columns are changed by the `dtypes` argument)

Reply:
All the columns were supposed to be int64, but I think it makes sense to be more explicit. I have forced in the last update some of the columns to be float64 and some other int32